### PR TITLE
builtinExtension-insiders to builtinExtension

### PR DESCRIPTION
**Pull request by MaddyDev**
Replacing the builtinExtension-insiders with builtinExtension since we moved the extension details to stable. Missed this update in my previous insidersTostable commit.

### CODE REVIEW

**Comment by MaddyDev** [comment]
We need to discuss how to handle scenarios when we have extensions both in builtinExtension.json and builtinExtension-insiders.json. Since the current code can only get content from either of them.

**Comment by aaomidi** [comment]
Whats the goal with this PR?

**Comment by anthonydresser** [comment]
It pulls in an external extension to be packaged with ads similar to how vsocde handle their node-debug extension's

**Comment by kevcunnane** [comment]
Agreed - it looks like I made a mistake not implementing the VSCODE_QUALITY check in this location too. We should either:
- Always include things from builtInExtensions.json for all quality streams, OR
- Fix to check quality and pull in the correct JSON file. I think this is the right change even though it means updating version in 2 places instead of 1. The benefit is, we can have a newer version of the extension pulled in via builtinExtensions-insiders.json and can validate it's working.

So, please update to use matching logic per Charles's suggestion

**Comment by MaddyDev** [comment]
> Agreed - it looks like I made a mistake not implementing the VSCODE_QUALITY check in this location too. We should either:
> 
> * Always include things from builtInExtensions.json for all quality streams, OR
> * Fix to check quality and pull in the correct JSON file. I think this is the right change even though it means updating version in 2 places instead of 1. The benefit is, we can have a newer version of the extension pulled in via builtinExtensions-insiders.json and can validate it's working.
> 
> So, please update to use matching logic per Charles's suggestion

Agreed, Me and Chris discussed all this offline before I sent out the PR, I thought of discussing with you and sending another PR on how we decide to go. Thanks for your feedback, I'll make those changes and will send out another commit.

File: build/lib/builtInExtensions.js [hunk 1]
```diff
@@ -16,10 +16,11 @@ const vfs = require('vinyl-fs');
 const ext = require('./extensions');
 const fancyLog = require('fancy-log');
 const ansiColors = require('ansi-colors');
+const product =  require('vs/platform/product/common/product');
```
Comment (Charles-Gagnon) [inline build/lib/builtInExtensions.js]: Did you test this? I'm not sure it's going to work - this is ran separately as a build-time task. Not within the ADS process.

https://github.com/microsoft/azuredatastudio/blob/7ebd1eb053c479a35a4b046fa00bbf5c6526aa99/scripts/sql.bat#L21

Comment (MaddyDev) [reply build/lib/builtInExtensions.js]: So reading the process.env as discussed offline now. Thanks!

**Pull request merged**